MyWeight/Extensions/UITableViewCell+ReuseIdentifier.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UITableViewCell+ReuseIdentifier.swift
+//  MyWeight
+//
+//  Created by Bruno Koga on 22/06/16.
+//  Copyright © 2016 Diogo Tridapalli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
```

Comment:
I think this is a behaviour we want to give to all the TableView and TableViewCells for free. Why wouldn't you want a defaulReusetIdentifier? And if a UITableViewCell subclass don't want/need to use it, just don't use it.
I also think it would make the code for the UITableViewDelegate/DataSource methods a little bit uglier, since we would need to check for the protocol conformance for each cell.